server/src/test/java/org/spine3/server/aggregate/AggregatePartShould.java

```diff
@@ -138,31 +178,96 @@ public void update_valid_entity_state() {
     /*
      Test environment classes
     ***************************/
-
     private static class AnAggregateRoot extends AggregateRoot<String> {
         protected AnAggregateRoot(BoundedContext boundedContext, String id) {
             super(boundedContext, id);
         }
     }
 
-    private static class WrongAggregatePart extends AggregatePart<String,
-            StringValue,
-            StringValue.Builder,
-            AnAggregateRoot> {
+    private static CommandEnvelope env(Message commandMessage) {
+        return CommandEnvelope.of(factory.command().create(commandMessage));
+    }
+
+    private void setAggregatePart() {
+        final AddTask addTask =
+                ((AddTask.Builder) Sample.builderForType(AddTask.class))
+                        .setProjectId(ProjectId.getDefaultInstance())
+                        .build();
+        taskPart.dispatchForTest(env(addTask));
+    }
+
+    private static class WrongAggregatePart
+            extends AggregatePart<String, StringValue, StringValue.Builder, AnAggregateRoot> {
         @SuppressWarnings("ConstantConditions")
         // Supply a "wrong" parameters on purpose to cause the validation failure
         protected WrongAggregatePart() {
             super(null);
         }
     }
 
-    private static class AnAggregatePart extends AggregatePart<String,
-            User,
-            User.Builder,
-            AnAggregateRoot> {
+    private static class AnAggregatePart
+            extends AggregatePart<String, User, User.Builder, AnAggregateRoot> {
 
         protected AnAggregatePart(AnAggregateRoot root) {
             super(root);
         }
     }
+
+    private static class TaskPart
+            extends AggregatePart<String, Task, Task.Builder, AnAggregateRoot> {
+
+        private TaskPart(AnAggregateRoot root) {
+            super(root);
+        }
+
+        @Assign
+        TaskAdded handle(AddTask msg) {
+            final TaskAdded result = TaskAdded.newBuilder()
+                                              .build();
+            return result;
```

Comment:
It's better not to leave the event message empty if there is at least one field. This may seem to be an error when debugging this tests.